entrypoints/popup/style.css

```
@@ -69,6 +69,14 @@ h1 {
     line-height: 1.5;
 }
 
+code {
+    font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)